sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/SettingLDEPFCsvHelper.java

```diff
@@ -39,6 +40,7 @@ public List<LDEPFSetting> getSettings() {
     }
 
     Function<LDEPFSettingBind, LDEPFSetting> lDEPFSettingsFunction = (LDEPFSettingBind setting) -> LDEPFSetting.builder()
```

Comment:
`parseRows CsvUtils` method is medeled within `ColumnPositionMappingStrategy`, it can only mapped to classes which contains `CsvBindByPosition` annotation.

https://github.com/com-pas/compas-sct/blob/develop/sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/CsvUtils.java#L54

Reply:
Yes and ?  
What is the problem with having `CsvBindByPosition` decorator ?

Reply:
`opencsv`  is just an example on how to read files. 
I used to separate it from the implementation of the Functional Interface, which had her own model and can be implemented by others tools not only `opencsv` which enforce  decorator like `CsvBindByPosition`